src/components/CopyContentButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import { ContentCopy } from '@mui/icons-material';
+import { Alert, IconButton, Snackbar } from '@mui/material';
+
+type Props = {
+	content: string;
+};
+
+export const CopyContentButton = ({ content }: Props) => {
+	const [open, setOpen] = useState(false);
+
+	const handleClick = () => {
+		setOpen(true);
+		navigator.clipboard.writeText(content);
+	};
+
+	return (
+		<>
+			<IconButton onClick={handleClick} color="primary">
+				<ContentCopy />
+			</IconButton>
+			<Snackbar
```

Comment:
Později bychom pro to možná vytvořit nějakou samostatnou komponentu a notifikovat tak uživatele o různých akcích (např. o vytvoření hádanky, uložení nové konfigurace hádanky apod.)

Reply:
Souhlas